CONTRIBUTING.md

```diff
@@ -99,25 +110,44 @@ Rebasing leads to a cleaner revision history which is much easier to follow and
+3.  You need to fetch the latest changes from eXist-db's GitHub: `git fetch upstream`. This will not yet change your local branches in any way.
 
-4.  You should first sync your `develop` branch with eXist's `develop` branch. As you always work in feature branches, this should a simple fast-forward by running: `git checkout develop` and then `git rebase upstream/develop`.
+4.  You should first sync your `develop` branch with eXist-db's `develop` branch. As you always work in feature branches, this should a simple fast-forward by running: `git checkout develop` and then `git rebase upstream/develop`.
     1.  If all goes well in (4) then you can push your `develop` branch to your remote server (e.g. GitHub) with `git push origin develop`.
 
-5.  You can then replay your work in your feature branch `my-feature` atop the lastest changes from the develop branch by running: `git checkout feature/my-feature` and then `git rebase develop`.
+5.  You can then replay your work in your feature branch `feature/my-feature` atop the lastest changes from the `develop` branch by running: `git checkout feature/my-feature` and then `git rebase develop`.
     1.  Should you encounter any conflicts during (5) you can resolve them using `git mergetool` and then `git rebase --continue`.
     2.  If all goes well in (5), and take care to check your history is correct with `git log`, then you can force push your `feature/my-feature` branch to your remote server (e.g. GitHub) with `git push -f origin feature/my-feature`. *NOTE* the reason you need to use the `-f` to force the push is because the commit ids of your revisions will have changed after the rebase.
 
 Note that it is worth syncing your branches that you are working on relatively frequently to prevent any large rebases which could lead to resolving many conflicting changes where your branch has diverged over a long period of time.
 
+## Tools
+Some developers may find that GitFlow tools can help them follow the above branching model. One such tool which may help is the [AVH Edition of GitFlow tools](https://github.com/petervanderdoes/gitflow).
+
+If you're not familiar with GitFlow, check out some of the good tutorials linked in ["Getting Started"](https://github.com/petervanderdoes/gitflow#getting-started) of the GitFlow AVH Edition page. There's also a very good [git-flow cheatsheet](http://danielkummer.github.io/git-flow-cheatsheet/).
 
-## Our `git-flow init` settings
+If you wish to contribute, the general approach using GitFlow AVH Edition is:
+
+-   Fork the repo on GitHub
+-   `git clone` your fork
+-   Make sure you've [GitFlow AVH Edition](https://github.com/petervanderdoes/gitflow) installed
+-   Run `git flow init` on the cloned repo using [these settings](#our-git-flow-init-settings).
+-   Use Git Flow to *start* a hotfix or feature i.e. `git flow feature start my-feature`.
+-   Do your stuff! :-)
+-   Commit to your repo. We like small, atomic commits that don't mix concerns.
```

Comment:
This point should perhaps be moved above to the "Pull Requests and Code Review" section. It's true regardless of whether someone uses the "AVH Edition of GitFlow tools."

Reply:
I am not sure that would work as this is specifically about how to use AVH GitFlow.

Reply:
Is AVH GitFlow used by anyone contributing to this project?